### PR TITLE
docs: Added a video introduction to Kaniko

### PR DESCRIPTION
**Pull request by vfarcic**

### CODE REVIEW

**Comment by vfarcic** [comment]
@googlebot I signed it!

**Comment by vbehar** [comment]
ahah @vfarcic you are everywhere! ;-)

**Comment by vfarcic** [comment]
@vbehar Yeah. Those videos are my weekend (outside office hours) pet projects.

Do you know what I need to do to merge this one?

**Comment by vbehar** [comment]
ping @tejal29 - or ask on the kaniko channel in the kubernetes slack

File: README.md [hunk 1]
```diff
@@ -116,6 +116,8 @@ After each command, we append a layer of changed files to the base image (if the
 
 ## Tutorial
 
+For a quick quick hands-on introduction and example, please watch [Kaniko - Building Container Images In Kubernetes Without Docker](https://youtu.be/EgwVQN6GNJg).
+
 For a detailed example of kaniko with local storage, please refer to a [getting started tutorial](./docs/tutorial.md).
 
 ## Using kaniko
```
Comment (tejal29) [inline README.md:123]:
```suggestion
For a detailed example of kaniko with local storage, please refer to a [getting started tutorial](./docs/tutorial.md).

Please see [References](#References) for more docs & video tutorials

## References

* [Kaniko - Building Container Images In Kubernetes Without Docker](https://youtu.be/EgwVQN6GNJg).


## Using kaniko
```

**Review by tejal29 — COMMENTED** [review]
Add a new references section

**Pull request merged**